src/invocation/InvocationService.ts

```diff
@@ -172,7 +172,7 @@ export class Invocation {
     }
 
     notify(clientMessage: ClientMessage): void {
-        assert(clientMessage != null, 'Response can not be null');
+        assertCond(clientMessage != null, 'Response can not be null');
```

Comment:
I feel like we don't even need a conditional check here. Looking at the code, it is not possible to pass a null client message here